cpp/arcticdb/version/local_versioned_engine.hpp

```diff
@@ -325,6 +327,10 @@ class LocalVersionedEngine : public VersionedEngine {
         const WriteOptions& write_options
     );
 
+    StreamId symbol_cache_id() {
+        return "__symbol_cache__";
```

Comment:
Why not call this `__symbol_stats[_cache]__` just to be more specific and more consistent with the method writing it?

Reply:
This is consistent with the pattern that we use elsewhere. Also, following Sausurre, we should recognise the arbitrary, conventional relation of signifier and signified - no-one's actually going to read it.